main/process-tx.zkasm

```diff
@@ -313,7 +314,7 @@ deploy:
         A => E
         ; check address to deploy is empty
                                        :CALL(isEmptyAccount)
```

Comment:
We should be able to deploy on an address that has some balance.
Here we are checking that the new address is totally empty meaning that `balance == nonce == code == 0x`.
I think in case of deployment we should only check that `nonce == code == 0x`